src/sentry/static/sentry/app/api.tsx

```diff
@@ -5,53 +5,52 @@ import Cookies from 'js-cookie';
 import isUndefined from 'lodash/isUndefined';
 
 import {openSudo, redirectToProject} from 'app/actionCreators/modal';
+import {CSRF_COOKIE_NAME, EXPERIMENTAL_SPA} from 'app/constants';
 import {
   PROJECT_MOVED,
   SUDO_REQUIRED,
   SUPERUSER_REQUIRED,
 } from 'app/constants/apiErrorCodes';
-import ajaxCsrfSetup from 'app/utils/ajaxCsrfSetup';
 import {metric} from 'app/utils/analytics';
 import {run} from 'app/utils/apiSentryClient';
+import getCookie from 'app/utils/getCookie';
 import {uniqueId} from 'app/utils/guid';
 import createRequestError from 'app/utils/requestError/createRequestError';
 
-import {EXPERIMENTAL_SPA} from './constants';
-
 export class Request {
   /**
    * Is the request still in flight
    */
   alive: boolean;
-  xhr: JQueryXHR;
+  /**
+   * Promise which will be resolved when the request has completed
+   */
+  requestPromise: Promise<Response>;
+  /**
+   * AbortController to cancel the in-flight request
+   */
+  aborter: AbortController;
 
-  constructor(xhr: JQueryXHR) {
-    this.xhr = xhr;
+  constructor(requestPromise: Promise<Response>, aborter: AbortController) {
+    this.requestPromise = requestPromise;
+    this.aborter = aborter;
     this.alive = true;
   }
 
   cancel() {
     this.alive = false;
-    this.xhr.abort();
+    this.aborter.abort();
     metric('app.api.request-abort', 1);
   }
 }
 
 /**
  * Setup the CSRF + other client early initalization.
+ *
+ * TODO(epurkhiser): This can be removed now that we are using fetch
  */
 export function initApiClient() {
-  jQuery.ajaxSetup({
-    // jQuery won't allow using the ajaxCsrfSetup function directly
-    beforeSend: ajaxCsrfSetup,
-    // Completely disable evaluation of script responses using jQuery ajax
-    // Typically the `text script` converter will eval the text [1]. Instead we
-    // just immediately return.
-    // [1]: https://github.com/jquery/jquery/blob/8969732518470a7f8e654d5bc5be0b0076cb0b87/src/ajax/script.js#L39-L42
-    converters: {
```

Comment:
`fetch` doesn't do anything crazy like jQuery was doing here. So we can drop this.